trade-finance-manager-api/src/v1/controllers/update-portal-deal-status.js

```diff
@@ -31,7 +31,7 @@ const updatePortalDealStatus = async (deal) => {
   }
 
   if (dealType === CONSTANTS.DEALS.DEAL_TYPE.BSS_EWCS) {
-    await api.updatePortalBssDealStatus(_id, status);
+    await api.updatePortalBssDealStatus({ dealId: _id, status, auditDetails });
   } else if (dealType === CONSTANTS.DEALS.DEAL_TYPE.GEF) {
     await api.updatePortalGefDealStatus(_id, status);
```

Comment:
did I miss this in a different PR or is this missing?

Reply:
Or is it just not technically called here, I feel like maybe we should add it in here anyway even if this route is unused as it's a bit weird otherwise

Reply:
Yeah -- this actually goes off to GEF. Will raise another PR for this